sopel_waifu/waifu.json

```diff
@@ -1238,34 +1238,87 @@
     ],
     "Super Sonico The Animation": [
         "Super Sonico",
-        "Super Pochaco"
```

Comment:
Instead of deleting Pochaco, why not just use "Super Sonico" as the franchise and include the games?

Reply:
I didn't know it was from a game. Isn't it a manga? That's the source on AL at least.

Reply:
It's a whole franchise. Sonico started as a Nitroplus mascot character, but from that seed a whole collection of games, manga, and anime sprouted.

Reply:
FWIW, please don't feel obligated to go through the whole Sonico franchise. I'm only asking that Pochaco not be deleted in this DB patch, and suggesting removal of "The Animation" from the franchise name.

I'm willing to fill out this category myself later.

Reply:
Implemented that strategy in another line note, since GH won't allow suggestions on deleted lines.